explanations/machine-learning-terms.md

```diff
@@ -25,6 +25,22 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>One example is working with words in Natural Language Processing. Words are usually transformed to a list of numbers  (embedded) before given as input to a neural network. The exact mapping between word and the list of numbers will be automatically learned.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Principal Component Analysis (PCA)</td>
+<td>
+    <p>An algorithm that is used to narrow down datasets.</p>
```

Comment:
This is true, but maybe you can make it more intuitive. I like that you didn't put "dimensionality reduction" in the beginning since it may be confusing.

I would go with something like "An algorithm that reduces the amount of data in a dataset leaving only the most important parts".

Reply:
What about "An algorithm that gets rid of the less important data in a dataset."?

Reply:
Maybe "An algorithm that removes less important data from a dataset."?

Reply:
That sound pretty similar to me, but I'm not fluent enough in English to see the difference ;)
So, let's go with this one.